spec/services/mentions_create_all_service_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe MentionsCreateAllService do
+  let(:user)        { create(:user) }
+  let(:article)     { create(:article, user_id: user.id) }
+  let(:comment)     { create(:comment, user_id: user.id, commentable_id: article.id) }
+  let(:comment2)    do
+    create(
+      :comment,
+      body_markdown: "Hello @#{user.username}, you are cool.",
+      user_id: user.id,
+      commentable_id: article.id,
+    )
+  end
+
+  it "creates mention if there is a user mentioned" do
+    comment.body_markdown = "Hello @#{user.username}, you are cool."
+    comment.save
+    MentionsCreateAllService.call(comment)
```

Comment:
`MentionsCreateAllService` can be replaced with `described_class`. It will make a test independent of the class name (e.g. if we'll need to change it)